streams/src/main/java/org/apache/kafka/streams/processor/internals/namedtopology/RemoveNamedTopologyResult.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.processor.internals.namedtopology;
+
+import org.apache.kafka.clients.admin.DeleteConsumerGroupOffsetsResult;
+import org.apache.kafka.common.KafkaFuture;
+import org.apache.kafka.common.internals.KafkaFutureImpl;
+
+import java.util.Objects;
+
+public class RemoveNamedTopologyResult {
+    private final KafkaFuture<Void> removeTopologyFuture;
+    private final DeleteConsumerGroupOffsetsResult deleteOffsetsResult;
+
+    public RemoveNamedTopologyResult(final KafkaFuture<Void> removeTopologyFuture, final DeleteConsumerGroupOffsetsResult deleteOffsetsResult) {
+        this.removeTopologyFuture = removeTopologyFuture;
+        this.deleteOffsetsResult = deleteOffsetsResult;
```

Comment:
I don't use the delete offset in this PR. I can remove it from the interface for now but I would rather leave it and just follow up